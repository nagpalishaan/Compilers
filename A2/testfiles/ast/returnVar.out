Program
  List
    Function
      IdDecl ID="n"
      List
      List
        DeclareVar
          IdDecl ID="a"
          Opt
            Numeral NUMERAL="5"
        Return
          IdUse ID="a"
