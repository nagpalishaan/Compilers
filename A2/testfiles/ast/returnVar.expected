Program
  List
    Function
      IdDecl ID="n"
      List
        DeclareVar
          IdDecl ID="a"
          List
            Numeral NUMERAL="5"
        Return
          IdUse ID="a"
